lms/templates/problem.html

```diff
@@ -5,38 +5,106 @@
 %>
 
 <%namespace name='static' file='static_content.html'/>
-<h3 class="hd hd-2 problem-header">
+<h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
```

Comment:
Thank you, @muzaffaryousaf. Given how pervasive the usage of id is, I'm leaning towards not changing all id usages to short_id. Would you be OK changing the specific instances that you need to be short_id in your own PR?